src/test/scala/nl.knaw.dans.easy.deposit/authentication/SessionSpec.scala

```diff
@@ -129,13 +129,14 @@ class SessionSpec extends TestSupportFixture with ServletFixture with ScalatraSu
       uri = "/auth/login",
       params = Seq(("login", "foo"), ("password", "bar"))
     ) {
-      status shouldBe OK_200
-      body shouldBe "signed in"
-      header("Content-Type") shouldBe "text/plain;charset=UTF-8"
+      status shouldBe NO_CONTENT_204
+      body shouldBe ""
+      header("Content-Type") shouldBe "text/html;charset=UTF-8"
       header("Expires") shouldBe "Thu, 01 Jan 1970 00:00:00 GMT" // page cache
       header("REMOTE_USER") shouldBe "foo"
       val newCookie = header("Set-Cookie")
       newCookie should startWith("scentry.auth.default.user=")
+      newCookie shouldNot startWith("scentry.auth.default.user=;")
```

Comment:
@vesaakerman : this could be more self-documenting if you would use a regular expression. What you want to say, I guess, is that there must be a non-empty scentry-cookie.

Reply:
that is correct.  I changed it now to a regular expression.